<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddEditAdrenalineNode</class>
 <widget class="QDialog" name="AddEditAdrenalineNode">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>435</width>
    <height>368</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add/Edit Syndicate Node</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QLineEdit" name="aliasLineEdit">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLineEdit" name="donationaddressLineEdit"/>
   </item>
   <item row="9" column="0">
    <widget class="QPushButton" name="okButton">
     <property name="text">
      <string>&amp;OK</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 28pt &quot;Baumans&quot;;
text-decoration: underline;
</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:26pt; text-decoration: underline;&quot;&gt;Masternode Configuration&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLineEdit" name="donationpercentageLineEdit"/>
   </item>
   <item row="4" column="0">
    <widget class="QLineEdit" name="txhashLineEdit"/>
   </item>
   <item row="10" column="0">
    <widget class="QPushButton" name="cancelButton">
     <property name="text">
      <string>&amp;Cancel</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLineEdit" name="addressLineEdit">
     <property name="frame">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLineEdit" name="privkeyLineEdit"/>
   </item>
   <item row="5" column="0">
    <widget class="QLineEdit" name="outputindexLineEdit"/>
   </item>
   <item row="8" column="0">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
